### DISCUSSION TITLE
Not redirecting to billing package v18

### DISCUSSION THREAD

**Discussion by slashcart** [Kyon147/laravel-shopify#122] [General]
It keeps showing skeleton

I am using tokenRoute for billing redirection as shown in image below

![image](https://user-images.githubusercontent.com/113967682/226269408-b6e045d6-505c-4ee3-98ba-6d2dcd3038fc.png)

Package Version: 18
Laravel Version: 10
PHP Version: 8.1
Template Engine: Blade

**Comment by zeeshanhaider6236dev** [comment]
F

**Comment by Kyon147** [comment]
Please don't open multiple threads for the same issue, as you've also opened up #121.
